src/d_loop.c

```diff
@@ -728,7 +728,7 @@ void TryRunTics (void)
 
     // [AM] If we've uncapped the framerate and there are no tics
     //      to run, return early instead of waiting around.
-    #define return_early (uncapped && counts == 0 && leveltime > oldleveltime)
+    #define return_early (uncapped && counts == 0)
```

Comment:
This code is correct, but just to avoid confusion, it's not always 60 fps in the menus. Removing `leveltime > oldleveltime` simply allows the menus to run at the same framerate as the rest of the game. That's what I wanted, so that's okay. Between `uncapped`, `fpslimit`, `use_vsync`, and the video driver settings, there could be a bunch of different scenarios but they're all fine to me.